File: 02. C# Part2/06. StringsАndTextProcessing-Homework/25. ExtractTextFromHTML/ExtractTextFromHTML.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;

//Write a program that extracts from given HTML file its title (if available), and its body 
//text without the HTML tags.

    class ExtractTextFromHTML
    {
        static void Main()
        {
            string input = "<html><head><title>News</title></head><body><p><a href=\"http://academy.telerik.com\">Telerik Academy</a>aims to provide free real-world practicaltraining for young people who want to turn into skilful .NET software engineers.</p></body></html>";
            int startText = input.IndexOf("<p>");
            int endText = input.IndexOf("</p>");
            string text = input.Substring(startText + 3, endText - startText - 3);
            string finalText = Regex.Replace(text, @"<a href=""(.*?)"">(.*?)</a>", "$2 ");
            int startTitle = input.IndexOf("<title>");
            int endTitle = input.IndexOf("</title>");
            string title = input.Substring(startTitle + 7, endTitle - startTitle - 7 );
            Console.WriteLine("Title: {0}", title);
            Console.WriteLine("Text: {0}", finalText);
        }
    }

File: 06. JavaScript Basics/09. Strings-Homework/09. Strings-Homework/06. ExtractTextFromHTML/ExtractTextFromHTML.js
//Write a function that extracts the content of a html page given as text.
//The function should return anything that is in a tag, without the tags.

function extractText() {
    var input = "<html><head><title>Sample site</title></head><body><div>text<div>more text</div>and more...</div>in body</body></html>;";
    var result = input.replace(/<(.*?)>/g, '');

    jsConsole.writeLine(input);
   jsConsole.writeLine(result);
}

extractText();